### DISCUSSION TITLE
Better way to indent gmt examples: use "if gmt"

### DISCUSSION THREAD

**Discussion by jidanni** [GenericMappingTools/gmt#7614] [Documentation development discussion]
In e.g., the cookbook, we often see examples indented like
```
gmt begin GMT_-R
	gmt set GMT_THEME cookbook
	gmt set MAP_FRAME_TYPE PLAI
        ...
gmt end show
```

Well, I have found a way to achieve similar indentation, that is
consistent with all shell script formatters. My new indentation method
can survive multiple runs through such pretty programs intact.

The above should be rewritten:
```
if gmt begin GMT_-R
then
   	gmt set GMT_THEME cookbook
	gmt set MAP_FRAME_TYPE PLAI
        ...
        gmt end show
fi
```
In fact my solution is so brilliant I think the entire cookbook should
be rewritten using it!

Tested with emacs' Shell-script mode using C-x h TAB.

Anyway: no need to teach the formatters anything special. It just works. Also it is good for you:
yes, you could do "set -ue" to catch more errors, but if `gmt` fails, ~~there is no point~~ 
it is just plain dangerous doing the stuff between the `then`...`fi` anyway.

**Comment by joa-quim** [comment]
> if gmt begin GMT_-R
then

Absolutely **NO WAY**. GMT scripts are supposed to run on Windows as well.
